javatests/arcs/sdk/wasm/CombineUpdatesTest.kt

```diff
@@ -110,18 +107,7 @@ class CombineUpdatesTest : AbstractCombineUpdatesTest() {
             handles.handle8,
             handles.handle9,
             handles.handle10
-        ) { e1, c, e3, e4, e5, e6, e7, e8, e9, e10 ->
-            val num1 = e1?.num
-            val num3 = e3?.num3
-            val num4 = e4?.num4
-            val num5 = e5?.num5
-            val num6 = e6?.num6
-            val num7 = e7?.num7
-            val num8 = e8?.num8
-            val num9 = e9?.num9
-            val num10 = e10?.num10
```

Comment:
The notion was this tests the typing. Not sure if it's useful or not?

Reply:
But how does it test the typing? doing `val num1 = e1?.num` doesn't test anything, it just introduces un unused variable with type inferred from the expression.

Reply:
It tests that num exists on e1, and num3 on e3. This means the right types are getting passed into the lambda.

Reply:
Oh, I see. It looked like an oversight, as no other tests had such structure and IDE highlighted this whole block as a block of unused variables. That's a good place to put a comment, as this code was indeed doing something useful, but it was not obvious what it does :)

Should I restore it in some form?